app/coffee/EmailHandler.coffee

```diff
@@ -0,0 +1,37 @@
+Settings = require("settings-sharelatex")
+logger = require("logger-sharelatex")
+
+EmailBuilder = require("./EmailBuilder")
+EmailSender = require("./EmailSender")
+
+module.exports = EmailHandler = {
+	sendOneTimeLoginEmail: (email, token, callback) ->
+		logger.info({ email }, "Sending one-time login message")
+		templateVars = {
+			title: "Overleaf login"
+			message: """
+The link below will allow you to log in once to the Overleaf maintenance site.
+
+Once logged in, you'll be able to download a read-only copy of your projects
+while we bring the site back.
```

Comment:
The copy is good already, but for your consideration:

> The link below will log you into Overleaf's read only maintenance site, so you can download your projects.

> The link is valid for one hour or until the maintenance is finished.

(This suggests an optional 'nice touch' in the main application: also add this route to web and put up a page (ignoring the token) explaining that the site is back and they can log in normally.)